.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/*
```

Comment:
The `.gitignore` file should not be inside `.idea`.  And `.idea` should be removed from Git.

Reply:
I cannot request a review from you
About generating the package.xml, I did it with a command `tree -X`. It's not fully automated because I still have to copy the output and paste it in package.xml. But I was told to leave it like that.